rsmtool/utils/wandb_logging.py

```diff
@@ -0,0 +1,72 @@
+"""
+Utility classes and functions for logging to Weights & Biases.
+
+:author: Tamar Lavee (tlavee@ets.org)
+"""
+import wandb
+
+EXCLUDE_WANDB_LOG = ["confMatrix"]
```

Comment:
Are we excluding this because it's already in the report?

Reply:
Wandb throws an exception when logging it, I think it's because it has an empty cell at 0,0. I will look into it this week but didn't want to waste too much time on it before I finish the integration.